src/Mod/Mod.cpp

```diff
@@ -5021,6 +5021,7 @@ void Mod::ScriptRegister(ScriptParserBase *parser)
 	mod.add<&Mod::getMaxDarknessToSeeUnits>("getMaxDarknessToSeeUnits");
 	mod.add<&Mod::getMaxViewDistance>("getMaxViewDistance");
 	mod.add<&getSmokeReduction>("getSmokeReduction");
+	mod.add<&Mod::getUnit>("getRuleUnit");
```

Comment:
This function have wrong API: `Unit * getUnit(const std::string &name, bool error)`, we should drop `bool` param and add `const` to returned object.